.github/workflows/test-build.yml

```diff
@@ -343,7 +376,7 @@ jobs:
           EMAIL_FOOTER_IMAGE_PATH: /config/logos/STFC-Logo-small.png
           EXTERNAL_AUTH_LOGIN_URL: http://localhost:9003/auth/Login.aspx
           EXTERNAL_AUTH_LOGOUT_URL: http://localhost:9003/auth/Login.aspx
-          EXTERNAL_AUTH_SERVICE_URL: http://localhost:1080/ws/UserOfficeWebService?wsdl
+          EXTERNAL_UOWS_API_URL: http://localhost:1080/users-service
```

Comment:
Also l may have missed but this port might be different for `http://localhost:1080/users-service`